cekit-cache-server/pom.xml

```diff
@@ -52,11 +52,6 @@
       <artifactId>jackson-dataformat-yaml</artifactId>
       <version>${jackson.version}</version>
     </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.datatype</groupId>
-      <artifactId>jackson-datatype-jsr310</artifactId>
-      <version>${jackson.version}</version>
-    </dependency>
```

Comment:
Is this dependency safe to remove because it is already brought in by the quarkus-bom?

Reply:
IIRC this dep was needed with the previous jackson version, seems it is not needed anymore with the version 2.9.9

Reply:
Okay, cool thx.